src/app/shared/issue-tables/issue-tables.component.html

```diff
@@ -143,7 +143,8 @@
       </button>
       <button mat-button color="warn" *ngIf="permissions.isIssueDeletable() && !issuesPendingDeletion[issue.id]
           && this.isActionVisible(action_buttons.DELETE_ISSUE)"
-              (click)="deleteIssue(issue.id)" matTooltip="Delete this issue" style="transform: scale(0.8)">
+              (click)="openDeleteDialog(issue.id); $event.stopPropagation()" 
+              matTooltip="Delete this issue" style="transform: scale(0.8)">
```

Comment:
Maybe it would be better to put the styling within the css file. What do you think? :smile: @ducbinh2611

Reply:
I see what you mean, I think for now we can leave it as it is and we can discuss this in a separate issue! :+1: 

And as a result, extract out the styles in another PR!